stat_test.go

```diff
@@ -333,6 +333,32 @@ func TestHarmonicMean(t *testing.T) {
 	}
 }
 
+func ExampleCircularMean() {
+	x := []float64{.0, 2 * math.Pi, 4 * math.Pi, 6 * math.Pi, 8 * math.Pi}
```

Comment:
This is a good example to have, but you should also have one where the mean is non zero.